### PR TITLE
DOCS-2369: Combine TrueCommand VM Deploy Instructions Into Main Insta…

### PR DESCRIPTION
**Pull request by jervin777**
…ll article

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by Mrt134** [comment]
Looks good, pinging @beanpole135 for a quick review

**Comment by beanpole135** [comment]
Looks like the HTML within the boxes is not rendering on the preview site:
image

**Comment by beanpole135** [comment]
Looking at the code more closely, that `</div>` in the second box should be at the end of the first box instead.

File: content/TrueCommand/TCGettingStarted/Install/_index.md
Comment (Mrt134) [inline content/TrueCommand/TCGettingStarted/Install/_index.md:18]:
```suggestion

```

File: content/TrueCommand/TCGettingStarted/Install/_index.md
Comment (Mrt134) [inline content/TrueCommand/TCGettingStarted/Install/_index.md:27]:
```suggestion
### Debian
```

File: content/TrueCommand/TCGettingStarted/Install/_index.md
Comment (Mrt134) [inline content/TrueCommand/TCGettingStarted/Install/_index.md:38]:
```suggestion

### Alpine
```

File: content/TrueCommand/TCGettingStarted/Install/_index.md
Comment (Mrt134) [inline content/TrueCommand/TCGettingStarted/Install/_index.md:52]:
```suggestion

### Void
```

File: content/TrueCommand/TCGettingStarted/Install/_index.md
Comment (Mrt134) [inline content/TrueCommand/TCGettingStarted/Install/_index.md:63]:
```suggestion

```

**Review by Mrt134 — CHANGES_REQUESTED** [review]
Added a few suggestions to clean up the div errors @beanpole135 mentioned.

**Review by Mrt134 — APPROVED** [review]
Changes are addressed and this is ready for merge. I'm checking with Web team about setting up redirects or aliases for the vmdeploy URL that will be going offline before merging this one.

**Comment by Mrt134** [comment]
Opened https://jira.ixsystems.com/browse/DM-304 to notify the web team about the URL change and have approval to merge.

**Comment by Mrt134** [comment]
Link checker hit a temporary FreeBSD site outage - approved for merge.

**Pull request merged**
